src/coreclr/jit/emitriscv64.cpp

```diff
@@ -4207,272 +4207,276 @@ regNumber emitter::emitInsTernary(instruction ins, emitAttr attr, GenTree* dst,
 
         if (needCheckOv)
         {
-            if (ins == INS_addi || ins == INS_addiw)
+            // At this point andi/ori/xori are excluded by previous checks
+            assert(ins == INS_addi || ins == INS_addiw);
```

Comment:
Diff starting from here is terrible due to indentation changes. Commit message highlights what is really going on here.

Reply:
"Hide whitespaces" in Diff view settings helps with that.